code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -210,7 +210,7 @@
 	name = "Disguise"
 	mechanics_text = "Disguise yourself as the enemy. Uses plasma to move."
 	///the regular appearance of the hunter
-	var/old_appearance
+	var/mob/old_appearance
```

Comment:
lol no, it's not a mob

Reply:
Mob is the highest generic implementation of rendering layer though?

Reply:
no it's an appearance

Reply:
How do you suggest that I access the layer attribute of old appearance then?

Reply:
I changed it to atom, which I think is its type.